docs/cli/about-stacks.md

```diff
@@ -13,72 +13,96 @@ next:
-settings often, doing everything in one Terraform project could cause problems.
-This might lead to some parts of your infrastructure needing to be rebuilt, which means
-more downtime.
+## What are Terramate Stacks?
 
-Using stacks helps with this issue. Stacks let you only update the parts of
-your system that need changes, without affecting everything else. Be careful to
-make the stacks the right size to avoid unnecessary repetition.
+Terramate Stacks are Infrastructure as Code agnostic and focus on adding functionality and features to improve the
+**developer experience**, **productivity** and **scalability** in Infrastructure as Code projects at any scale.
 
-In short, try to keep code that changes for different reasons separate, so you
-don't create problems when making updates.
+You can use Terramate Stacks to manage IaC technologies such as Terraform, OpenTofu, Terragrunt, Kubernetes, AWS Cloud
+Formation, AWS Cloud Development Kit  (CDK), Bicep, and others.
 
-## Reduce the blast radius
+> **Note:** Some IaC technologies, such as AWS Cloud Development Kit (CDK) offer a native implementation of stacks,
+> while others don’t. It’s important to understand that Terramate integrates seamlessly with those approaches.
+> E.g., Terramate can be used to manage Terraform workspaces and CDK Stacks.
 
-A small change can lead to catastrophic events if you're not careful. For instance,
-forgetting a `prevent_destroy` attribute in the production database lifecycle
-declaration. Also, someone within your organisation or team may commit mistakes
-and it's better if you could reduce the impact of such errors.
-An extreme example is: avoiding a database instance destroy because of a DNS TTL
-change.
+Most of the time, Terramate projects manage *dozens*, *hundreds*, or even *thousands* of stacks. This is possible because Terramate CLI provides a neat range of features that allow you to create and manage stacks efficiently at any scale:
 
-## Reduce execution time
+- Stacks can be **created**, **cloned**, and **compared** with a single command.
+- Stacks can be **orchestrated and targeted** for operations, which allows to execute any command over a filtered section of stacks.
+- The **change detection** allows to execute stacks that contain changes only.
+- The **order of execution** of stacks can be configured explicitly in addition to the default order of execution.
+- You can **generate code** in stacks. E.g. you can generate the Terraform backend configuration for all Terraform stacks or a Kubernetes manifest to create a secret for all Kubernetes stacks that follow certain criteria.
+- Stacks can be used to **manage ownership** by leveraging concepts such as [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
+- Stacks allow you to implement **multi-**IaC and **multi-step** scenarios.
```

Comment:
```suggestion
- Stacks allow you to implement **multi-IaC** and **multi-step** scenarios.
```
as it was caused the *'s to be shown as *s rather than bold